### PR TITLE
(AzureCXP) fixes MicrosoftDocs/azure-docs#106119

### PR DESCRIPTION
**Pull request by mumurug-MSFT**
Updated Azure Status page based on https://learn.microsoft.com/en-us/azure/service-health/whats-new#azure-status-page-location-changed

# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@ecfan - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by mumurug-MSFT** [comment]
@ecfan Not sure how I can update the below two references: 
1. https://github.com/Azure/azure-quickstart-templates/blob/8220216205e1ca8e822bca87402b0ac64367960a/quickstarts/microsoft.logic/logic-app-create/README.md 

2. https://github.com/Azure/azure-quickstart-templates/blob/8220216205e1ca8e822bca87402b0ac64367960a/quickstarts/microsoft.logic/logic-app-create/azuredeploy.json

Please take a look or let me know so that I can submit accordingly.

File: quickstarts/microsoft.logic/logic-app-create/main.bicep [hunk 1]
```diff
@@ -2,7 +2,7 @@
 param logicAppName string
 
 @description('A test URI')
-param testUri string = 'https://status.azure.com/en-us/status/'
+param testUri string = 'https://azure.status.microsoft/en-us/status/'
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/main.bicep:5]:
```suggestion
param testUri string = 'https://azure.status.microsoft/status/'
```

Comment (ecfan) [reply quickstarts/microsoft.logic/logic-app-create/main.bicep]: @mumurug-MSFT: Thanks, use these links instead, which should have the edit button enabled. The original links were set to a specific commit, but the following links are set to the main branch:

- https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.logic/logic-app-create/README.md
- https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.logic/logic-app-create/azuredeploy.json

**Comment by azure-quickstarts** [comment]
@ecfan - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Pull request merged**